src/colour_output.rs

```diff
@@ -0,0 +1,37 @@
+use std::str::FromStr;
+
+use palette::Srgb;
+
+pub enum ColourOutput {
+    Hex,
+    Rgb255,
+    Rgb01,
+}
+
+impl ColourOutput {
+    pub fn format_colour(&self, colour: &Srgb<u8>) -> String {
+        match self {
+            Self::Hex => format!("#{:02x}{:02x}{:02x}", colour.red, colour.green, colour.blue),
+            Self::Rgb255 => format!("rgb({}, {}, {})", colour.red, colour.green, colour.blue),
+            Self::Rgb01 => format!(
+                "rgb({:.3}, {:.3}, {:.3})",
```

Comment:
Let's get the comment from the PR description in here explaining why you chose 3 decimal places.